packages/usa-date-picker/src/index.js

```diff
@@ -80,30 +80,15 @@ const CALENDAR_YEAR_FOCUSED = `.${CALENDAR_YEAR_FOCUSED_CLASS}`;
 
 const VALIDATION_MESSAGE = "Please enter a valid date";
 
-const MONTH_LABELS = [
-  "January",
-  "February",
-  "March",
-  "April",
-  "May",
-  "June",
-  "July",
-  "August",
-  "September",
-  "October",
-  "November",
-  "December",
-];
-
-const DAY_OF_WEEK_LABELS = [
-  "Sunday",
-  "Monday",
-  "Tuesday",
-  "Wednesday",
-  "Thursday",
-  "Friday",
-  "Saturday",
-];
+const MONTH_DATE_SEED = Array.from({ length: 12 }).map(
+  (_, i) => new Date(0, i)
+);
+
+const DAY_OF_WEEK_DATE_SEED = Array.from({ length: 7 }).map(
+  (_, i) => new Date(0, 0, i)
+);
+
+const CALENDAR_LABELS_BY_LANG = new Map();
```

Comment:
**thought (non-blocking)**: I can see the benefits of using a map, but it could lead to situations where we have several languages in a map if the document language gets updated dynamically.

**For example**
```js
// CALENDAR_LABELS_BY_LANG
{
  'en' => {…}, 
  'es' => {…}, 
  'en-uk' => {…}, 
  'de' => {…},
  …etc
}
```

Reply:
this is true, but it also means that if someone is changing a language back and forth for some reason we don't have to recalculate the langauge